examples/broker/src/broker_shell.cpp

```diff
@@ -77,6 +77,11 @@ int BrokerShell::Run(etcpal::Logger& log)
   broker_settings.dns.model = "RDMnet Broker Example App";
   broker_settings.listen_port = initial_data_.port;
 
+  if (!initial_data_.netints.empty())
```

Comment:
Why check for empty before making this assignment? If it's empty, the assignment will essentially be a no-op.

Reply:
I went ahead and removed the condition.